packages/data/src/index.js

```diff
@@ -142,6 +142,8 @@ export const suspendSelect = defaultRegistry.suspendSelect;
  */
 export const subscribe = defaultRegistry.subscribe;
 
+export const waitForTransition = defaultRegistry.waitForTransition;
```

Comment:
This export is wrong. `waitForTransition` is no longer a method of `defaultRegistry`, it's a utility reexported from `./utils`.

This is also the reason why `docs:build` doesn't "see" the doc block for the function.